iroh-gateway/src/client.rs

```diff
@@ -43,6 +50,12 @@ impl Client {
             .metrics
             .tts_file
             .set(start_time.elapsed().as_millis() as u64);
+        let n = res.len() as u64;
+        state
+            .metrics
+            .bitrate_out
```

Comment:
this is calculating a byte not a bitrate

Reply:
Whoops, bad copy pasta, fixing before merge